### ISSUE TITLE
Settings button isn't translated

### ISSUE THREAD

**Issue by artehe** [stayintarkov/SIT.Manager.Avalonia#45]
The settings button doesn't have any localisation attached, which we could resolve in the code behind the MainView but I would rather this was in xaml.

The best idea I've seen so far is to add this (or similar)
```xaml
<NavigationView.PaneFooter>
    <Button Background="Transparent" BorderThickness="1" HorizontalAlignment="Center"
            Height="40" Width="40"
            Click="SettingsButton_OnClick">
        <SymbolIcon Symbol="Setting"/>
    </Button>
</NavigationView.PaneFooter>
```

And set `IsSettingsVisible="False"` on the NavigationView, but this'll cuase a few other problems.

I'm open to opinions however.

**Comment by J0nathan550** [comment]
No need, I fixed that already

**Comment by J0nathan550** [comment]
Solved in #50

**Comment by J0nathan550** [comment]
Solved already in #50 @Plootie  :D
